[15:00:09][DEBUG][tests/trajector/conftest.py:33] <TestReport 'tests/trajector/test_manual.py::TestExample::test_existing_user_login_with_valid_password' when='setup' outcome='passed'>
[15:00:27][DEBUG][tests/trajector/conftest.py:33] <TestReport 'tests/trajector/test_manual.py::TestExample::test_existing_user_login_with_valid_password' when='call' outcome='failed'>
[15:00:27][DEBUG][tests/trajector/conftest.py:33] <TestReport 'tests/trajector/test_manual.py::TestExample::test_existing_user_login_with_valid_password' when='teardown' outcome='passed'>
